assembl/static2/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "build": "webpack --progress --colors",
+    "build": "rm -rf node_modules/react-i18nify/node_modules;webpack --progress --colors",
```

Comment:
Actually no, when you run `yarn` again, it doesn't put the dependencies again.
Just to explain why I did that,
react-i18nify package.json defines a fixed version of prop-types 15.5.8 and react 15.5.4, which is a bad practice for libraries and react-redux-i18n depends on a fixed version of react-i18nify.
So bundle.js contained a second version of react 18 KB.
I just did a PR for that https://github.com/JSxMachina/react-i18nify/pull/35
When there is a new release of react-i18nify and react-redux-i18n, we can remove this.